elasticreindexer/README.md

```diff
@@ -1,11 +1,11 @@
 # Elastic reindexer tool
 
-- This tool can be used to copy an Elasticsearch  index/indices from a cluster to another. It can also copy the indices 
-mappings ( we recommend to use it without copying the mappings ). The indices mappings part can be done with the `indices-creator` tool.
-Also, with the `indices-creator` tool some properties of the indices can be changed ( depending on the needs ).
+- This tool can be used to copy an Elasticsearch index/indices from a cluster to another. It can also copy the indices 
+mappings (we recommend to use it without copying the mappings because most often are not exposed). The indices mappings part can be done with the `indices-creator` tool.
```

Comment:
because most often **they** are not exposed

Reply:
done.